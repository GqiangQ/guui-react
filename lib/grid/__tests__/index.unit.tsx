import {configure} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Row, Col} from '../index';

configure({adapter: new Adapter()});

describe('Grid', () => {
  it('combines Row and Col', () => {
    const tree = renderer
      .create(
        <Row>
          <Col className="col" span={2}/> <Col className="col" span={2}/>
          <Col className="col" span={2}/> <Col className="col" span={2}/>
          <Col className="col" span={2}/> <Col className="col" span={2}/>
          <Col className="col" span={2}/> <Col className="col" span={2}/>
          <Col className="col" span={2}/> <Col className="col" span={2}/>
          <Col className="col" span={2}/> <Col className="col" span={2}/>
        </Row>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});
