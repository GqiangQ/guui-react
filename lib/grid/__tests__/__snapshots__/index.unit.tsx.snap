// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid combines Row and Col 1`] = `
<div
  className="gu-row gu-row-alignLeft "
  style={
    Object {
      "margin": undefined,
    }
  }
>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
  <div
    className="gu-col gu-col-span2"
    style={
      Object {
        "padding": undefined,
      }
    }
  >
    <div
      className="col"
    />
  </div>
</div>
`;
